lua/kubectl/views/init.lua

```diff
@@ -100,11 +100,24 @@ function M.Hints(headers)
   buffers.set_content(buf, { content = content, marks = marks })
 end
 
+local function merge_views(cached_resources, views_table)
```

Comment:
Great idea! Maybe we could move the function to definition.lua though?

Reply:
Done. sheesh that was exhausting calculating the cpu and mem...

Reply:
lol yeah I was like, I'll read that part later 🤣